packages/plugins/AddressSearch/README.md

```diff
@@ -25,7 +25,7 @@ In `categoryProperties` and `groupProperties`, id strings called `groupId` and `
 | fieldName | type | description |
 | - | - | - |
 | searchMethods | searchMethodsObject[] | Array of search method descriptions. Only searches configured here can be used. |
-| afterResultComponent | VueConstructor? | If given, this component will be rendered in the last line of every single search result. It will be forwarded its search result feature as prop `feature` of type `GeoJSON.Feature`, and the focus state of the result as prop `focus` of type `boolean`. |
+| afterResultComponent | VueConstructor? | If provided, this component will be rendered at the bottom of each search result. It will receive two props: `feature` of type `GeoJSON.Feature`, which represents the search result feature, and `focus` of type `boolean`, which indicates whether the result is focused. It is then passed as a Vue Component. |
```

Comment:
Not a fan of the paraphrasing as it adds no value. Also, "at the bottom of" is wrong as it indicates a placement below. The prior "in the last line" wasn't helpful either; it's just technically *after*. I suggest the following solution.

```suggestion
| afterResultComponent | VueConstructor? | If given, this component will be rendered after every single search result's title. It will be forwarded its search result feature as prop `feature` of type `GeoJSON.Feature`, and the focus state of the result as prop `focus` of type `boolean`. This component may e.g. be used to display further information about a result or offer additional interactions. Mind that you may override the default selection by providing a `customSelectResult`, allowing you to completely lift selection options to this component. Using this feature requires coding knowledge. |
```